.github/workflows/pkgci_regression_test.yml

```diff
@@ -74,6 +74,11 @@ jobs:
         with:
           name: linux_x86_64_release_packages
           path: ${{ env.PACKAGE_DOWNLOAD_DIR }}
+      - name: Install dependencies
+        if: ${{ inputs.artifact_run_id != '' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install gh
```

Comment:
Actually, I'll try https://github.com/actions4gh/setup-gh instead of apt.

Reply:
Setup actions to install `gh` (outside of apt) aren't working out of the box for me

For https://github.com/actions4gh/setup-gh, https://github.com/actions4gh/setup-gh/issues/7 (sample debug logs: https://github.com/iree-org/iree/actions/runs/13728123378/job/38399699167)

For https://github.com/ksivamuthu/actions-setup-gh-cli, I'm not sure... sometimes it succeeds but then reports `FileNotFoundError: [Errno 2] No such file or directory: 'gh'`: https://github.com/iree-org/iree/actions/runs/13728170101/job/38399321434?